tests/acceptance/features/bootstrap/WebUIPersonalSharingSettingsContext.php

```diff
@@ -80,4 +80,22 @@ public function switchAutoAcceptingFederatedShares($action) {
 			$this->getSession(), $action
 		);
 	}
+
+	/**
+	 * @Then User-based auto accepting checkbox should not be displayed on the personal sharing settings page in the webUI
+	 *
+	 * @return void
+	 */
+	public function autoAcceptingCheckboxShouldNotBeDisplayedOnThePersonalSharingSettingsPageInTheWebui() {
+		PHPUnit_Framework_Assert::assertFalse($this->personalSharingSettingsPage->isAutoAcceptLocalSharesCheckboxDisplayed());
```

Comment:
please make the line shorter, that will make it easier to review
```suggestion
		PHPUnit_Framework_Assert::assertFalse(
		    $this->personalSharingSettingsPage->isAutoAcceptLocalSharesCheckboxDisplayed()
		);
```